py_mini_racer/extension/mini_racer_extension.cc

```diff
@@ -580,7 +554,7 @@ static BinaryValue* MiniRacer_eval_context_unsafe(
     // NOTE: this is very important, we can not do an raise from within
     // a v8 scope, if we do the scope is never cleaned up properly and we leak
     if (!eval_result.parsed) {
-        result = ALLOC(BinaryValue);
+        xalloc(result);
```

Comment:
Even if unnecessary, wouldn't it be more readable to explicitly set result? Like on line 438? I'm assuming the compiler will remove the extraneous copy.
If you want to stick to the denser style, you should update lines 401 and 438 to make everything consistent.

Reply:
OK, I understand this can be confusing. I'll just return the pointer to the allocated memory and not pass the result by reference.